examples/basic_sprite.rs

```diff
@@ -37,13 +32,9 @@ fn setup(
         },
         text_position: CosmicTextPosition::Center,
         cosmic_attrs: CosmicAttrs(AttrsOwned::new(attrs)),
+        text: CosmicText::OneStyle("😀😀😀 x => y".to_string()),
```

Comment:
Let's be consistent and call it `set_text` until we don't sync a buffer content.

Reply:
yeah, anything that you think better, `text` in both_cases or `set_text`.